media/Config/buildDom.js

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// remove every options on DOM
+const emptyOptionBox = function(isImport) {
+  if (!isImport) {
+    // erase select tag for choosing framework
+    const locaForSelect = document.querySelector('#locaForSelect');
+    while (locaForSelect.hasChildNodes()) {
+      locaForSelect.removeChild(locaForSelect.firstChild);
+    }
+
+    const toolList = document.querySelectorAll('.tools div');
+    for (let i = 0; i < toolList.length; i++) {
+      toolList[i].classList.remove('selected');
+    }
+  }
+
+  // erase option names
+  const optionsName = document.querySelector('#optionsName');
+  while (optionsName.hasChildNodes()) {
+    optionsName.removeChild(optionsName.firstChild);
+  }
+
+  // erase option values
+  const optionsValue = document.querySelector('#optionsValue');
+  while (optionsValue.hasChildNodes()) {
+    optionsValue.removeChild(optionsValue.firstChild);
+  }
+};
+
+// make DOM disable depends on tool.use
+const chooseDisable = function(tool) {
+  const useBtn = document.querySelector('#useBtn');
+  const optionFieldset = document.querySelector('#options');
+
+  // when tools are related with one-import
+  if (tool.type.startsWith('one-import')) {
+    if (oneImport.use === true) {
+      useBtn.checked = true;
+      optionFieldset.disabled = false;
+
```

Comment:
Thanks for adding spaces between lines.
But I think this line can be removed as there are `}` or `{` that separate the logic.